test/aggregator_test.rb

```diff
@@ -77,4 +77,16 @@ def test_name_empty
     end
     assert_equal error.message, "Name can't be blank"
   end
+
+  def test_range
```

Comment:
Sorry, one more comment: this test will pass without `range`, so should probably be improved.

Reply:
I thought the `create_users` method would create a larger changeset and because of the `assert_equal` the test would fail, but I'll be honest, I didnt run it, had some issues building the native pg gem, let me get that fixed and I'll look to improve the test